common/metrics/defs.go

```diff
@@ -1265,21 +1265,23 @@ const (
 	SequentialTaskQueueProcessingLatency
 	SequentialTaskTaskProcessingLatency
 
+	HistoryArchiverArchiveNonRetryableErrorCount
+	HistoryArchiverArchiveTransientErrorCount
 	HistoryArchiverHistoryMutatedCount
-	HistoryArchiverTotalUploadSize
+	HistoryArchiverUploadedHistorySize
```

Comment:
rename: HistoryArchiverUploadedSize - like you pointed out this metric does not indicate history size it just indicates load on archiver, so no need to include history in metric name